map_gui/src/options.rs

```diff
@@ -55,7 +55,7 @@ impl Options {
             debug_all_agents: false,
 
             traffic_signal_style: TrafficSignalStyle::BAP,
-            color_scheme: ColorSchemeChoice::DayMode,
+            color_scheme: ColorSchemeChoice::Pregame,
```

Comment:
This effectively continues to use the old day mode in the 15min tool, parking mapper, OSM viewer, etc. That seems fine for now. I'll have a look this weekend and cut them over to the new day mode, under the assumption that we're getting rid of the legacy day theme once some of the cutscene stuff is cleared up.